pyproject.toml

```
@@ -0,0 +1,36 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.coverage.run]
+source = ["imaplib2/tests"]
+source_pkgs = ["imaplib2"]
+
+[tool.coverage.paths]
+source = ["imaplib2/", "*/lib/*/site-packages/imaplib2"]
```

```suggestion
source = [".", "*/lib/*/site-packages/imaplib2"]
```

---

This doesn't work on my machine. It only reports the coverage of `test_imports.py`. That list identifies which paths are equivalent. So I think it's either `["imaplib2/", "*/lib/*/site-packages/imaplib2"]` or `[".", "*/lib/*/site-packages"]`.

My understanding:
`*/lib/*/site-packages/imaplib2/` denotes the actual prefix of the source files being tested (i.e., those installed by `tox`), while `imaplib2/` indicates the original prefix of the source files relative to code repository. `coverage` then knows how to combine the results from different `tox` environments (that's why the placeholders) and presents the results using the paths relative to the code repo.

---

Ah in that case I think you want:

```suggestion
source = [
            ".",
            ".tox/*/lib/*/site-packages/",
            '.tox\\*\\Lib\\site-packages\\',
        ]
```

---

This works!